src/components/Arrow/Arrow.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from "react";
+import { Position } from "../../types/Position";
+import styles from "./Arrow.module.scss";
+import {
+  ArrowBody,
+  ArrowHead,
+  ArrowBodyVertical,
+  MirroredArrowHead,
+  ArrowHeadVertical,
+  MirroredArrowHeadVertical,
+} from "./ArrowParts";
+import { ArrowDirection, ArrowType } from "./Utils";
+
+export type ArrowProps = {
+  start: Position;
+  end: Position;
+  type: ArrowType;
+  direction: ArrowDirection;
+};
+
+export const Arrow: React.FC<ArrowProps> = ({
```

Comment:
Would be great if we could import this from `h5p-topic-map` or somehow share the code.

```suggestion
// TODO: Share code with h5p-topic-map instead of duplicating
export const Arrow: React.FC<ArrowProps> = ({
```